apps/kibana/templates/services/app/deploy.yml.j2

```
@@ -48,6 +48,10 @@ spec:
         - name: kibana
           image: "{{ kibana_image_name }}:{{ kibana_image_tag }}"
           imagePullPolicy: Always
+          command:
+            - "bash"
+            - "-c"
+            - "SERVER_UUID=$(uuidgen) /usr/local/bin/kibana-docker"
```

```suggestion
          command: ["/bin/bash"]
          args: ["-c", "SERVER_UUID=$(uuidgen) /usr/local/bin/kibana-docker"]
```

according to the [k8s documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#run-a-command-in-a-shell), it should be done like this

---

Wow, so there is a lot of trays that needs to be fixed!